src/test/java/com/google/api/codegen/testdata/python_README_library.baseline

```diff
@@ -68,8 +68,8 @@ LibraryServiceClient
 
 .. code:: py
 
-  from google.cloud.gapic.example.library.v1 import library_service_client
-  client = library_service_client.LibraryServiceClient()
+  from google.cloud.example import library_v1
+  client = library_v1.LibraryServiceClient()
```

Comment:
Possibly out of scope: Would it be possible to always name the class `Client` unless there is more than one `service` in the proto? (Yes, I know there is a risk about "what if a second client" comes along and I am willing to accept that risk.)

If that is hard because of the "if more than one" logic, feel free to demur.